src/main/java/Task.java

```diff
@@ -0,0 +1,46 @@
+import java.util.*;
+
+/**
+ * Each task will hold the status of done or not done, plus description of the
+ * task to be done.
+ */
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Call this method to see if we should mark X or leave blank.
+     * 
+     * @return
+     */
+    public String getStatusIcon() {
+        return (this.isDone ? "X" : " "); // mark done task with X
+    }
+
+    // Getters
+    public boolean getIsDone() {
+        return this.isDone;
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
```

Comment:
The use of verbs in the method names makes it clear what the method is doing... but maybe adding javadocs comment to describe the methods, would make it even more easier, I guess.... what do u think?